lib/experimental/modules/class-gutenberg-modules.php

```diff
@@ -29,6 +29,7 @@ class Gutenberg_Modules {
 	 *
 	 * @param string $module_identifier The identifier of the module. Should be unique. It will be used in the final import map.
 	 * @param string $src               Full URL of the module, or path of the script relative to the WordPress root directory.
+	 * @param string $usage             Specifies where the module would be used. Can be 'admin', 'frontend', or 'both'.
```

Comment:
I think the same script can be probably be used in both frontend and backend. I also wonder if we'd have more "usage" in the future. Do you think a string is enough or should it be an array? I'm not very opinionated, just asking the question I guess.

Reply:
Makes sense. I changed it so now it accepts a string or an array of strings: https://github.com/WordPress/gutenberg/pull/56092/commits/e0201cee783b9cbfe7b69b7de9b78404d7909327